metals/src/main/scala/scala/meta/internal/metals/ConfiguredLanguageClient.scala

```diff
@@ -93,7 +99,14 @@ final class ConfiguredLanguageClient(
       params: ExecuteCommandParams
   ): Unit = {
     if (config.executeClientCommand.isOn) {
-      underlying.metalsExecuteClientCommand(params)
+      params.getCommand match {
+        case _ if config.executeClientCommand.isOff =>
+        // ignore
+        case ClientCommands.RefreshModel.id if !debuggingSupported =>
```

Comment:
Does this work? I think there's an `unapply`
```suggestion
        case ClientCommands.RefreshModel() if !debuggingSupported =>
```